src/main/java/Application.java

```
@@ -0,0 +1,127 @@
+import controllers.AudioPlayerController;
+import model.*;
+
+import java.util.*;
+
+import static model.Genre.*;
+
+/**
+ * Starting point of the application.
+ * Initializing the data.
+ * Starting the audio player.
+ * Executing the listed methods after exiting the audio player.
+ */
+public class Application {
+    private static AudioPlayerController audioPlayerController;
+
+    public static void main(String[] args) {
+        init();
+
+        audioPlayerController.start();
+
+        System.out.println(audioPlayerController.getAudioPlayerInfo());
+
+        System.out.println("Input song title:");
+        System.out.println(audioPlayerController.getAuthorPositionInPlaylist());
+
+        System.out.println("Please input full Author name:");
+        List<Song> foundSongsOfAuthor = audioPlayerController.getSongsOfAuthor();
+
+        if (foundSongsOfAuthor != null && !foundSongsOfAuthor.isEmpty()) {
+            foundSongsOfAuthor.forEach(s -> System.out.println(s.getShortInfo() + "\n"));
+        } else {
+            System.out.println("Author not found.");
+        }
+
+        System.out.println("There are: " + audioPlayerController.getCountOfAllListedSongs() + " songs in the list.");
+    }
+
+    /**
+     * This initializes the application:
+     * - creates Objects - Authors, Singers, Songs;
+     * - enlists songs in the playlist for the AudioPlayer to work with;
+     */
+    private static void init() {
+        audioPlayerController = new AudioPlayerController();
+        Author johnDou = null;
+        Author koleKolev = null;
+
+        try {
+            johnDou = new Author("John Dou");
+        } catch (CustomException exception) {
+            System.out.println(exception.getMessage());
```

this is not a good practice to swallow the exception and continue, just let the exception stop the execution